frontend/src/pages/projects/screens/spawner/spawnerUtils.ts

```diff
@@ -161,9 +161,7 @@ export const isCompatibleWithAccelerator = (
     return false;
   }
 
-  return getCompatibleAcceleratorIdentifiers(obj).some(
-    (accelerator) => accelerator === acceleratorIdentifier,
-  );
+  return getCompatibleAcceleratorIdentifiers(obj).some((cr) => cr === acceleratorIdentifier);
```

Comment:
Weird to call it `cr` 🤔 But variable names are not the end of the world...

Reply:
oh yeah in that area it is weird i agree. should be something like _identifier_